plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeTransformTelemetryMetadataSingleton.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.model
+
+object CodeTransformTelemetryMetadataSingleton {
+    private val instance = CodeTransformTelemetryMetadata()
+
+    fun getInstance() = instance
```

Comment:
getInstance() in this project implies the platform "service" construct, rather than a singleton

Reply:
Are you saying the coding style for this repo of `getInstance` is reserved as a keyword for a "service" level construct? Or project level?

If its service level, then it would fit because this is a single instance shared across our service.

Reply:
service as in the intellij platform service construct